packages/cubejs-api-gateway/requestParser.js

```diff
@@ -0,0 +1,19 @@
+module.exports = (req, res) => {
+  const path = req.originalUrl || req.path || req.url;
+  const httpHeader = req.header && req.header('x-forwarded-for');
+  const ip = req.ip || httpHeader || req.connection.remoteAddress;
+  const requestData = {
+    path,
+    method: req.method,
+    status: res.statusCode,
+    ip,
+    time: (new Date()).toISOString(),
```

Comment:
There're also some pending changes for logging so it should be expected that timestamps will be handled in one place.